### PR TITLE
utils,ui: obfuscate sensitive log info, use POST for configureOutOfBandManagement

### LINKED ISSUE #8526: Password for IPMI user showing up in plain text in the log files
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
Log Files
~~~

##### CLOUDSTACK VERSION

~~~
4.18.1
~~~

##### OS / ENVIRONMENT

OS= Ubuntu 22.04, KVM, OOBM HPE ILO for HA enabled hosts 

##### SUMMARY

We setuo HA for our host and the OOBM IPMI users password is displayed in plain text when you tail or open the /var/log/cloudstack/management/ management-server.log

##### STEPS TO REPRODUCE

~~~

![image](https://github.com/apache/cloudstack/assets/148808916/498e073c-f3f6-474d-86b5-39e8404a449a)

we change the IP address and password for security purposes.
also, the IPMI user needs to be a full admin with privilege to change users accounts. is that expected?
~~~

##### EXPECTED RESULTS

~~~
for the password to be encrypted in the log files 
also, the IPMI user needs to be a full admin with privilege to change users accounts. is that expected?
~~~

##### ACTUAL RESULTS

~~~
The password is visible in plain text
~~~

### PR DESCRIPTION
**Pull request by shwstppr**
### Description

Fixes #8526

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by shwstppr** [comment]
@blueorangutan package

**Comment by shwstppr** [comment]
@blueorangutan test

**Comment by vishesh92** [comment]
@blueorangutan ui

File: utils/src/main/java/org/apache/cloudstack/utils/process/ProcessRunner.java [hunk 1]
```diff
@@ -74,27 +96,28 @@ public ProcessResult executeCommands(final List<String> commands, final Duration
         String stdError = null;
 
         String oneLineCommand = StringUtils.join(commands, " ");
+        String commandLog = removeCommandSensitiveInfoForLogging(oneLineCommand);
```
Comment (DaanHoogland) [inline utils/src/main/java/org/apache/cloudstack/utils/process/ProcessRunner.java:99]:
```suggestion
        String commandLog = removeCommandSensitiveInfoForLogging(StringUtils.join(commands, " "));
```

**Review by DaanHoogland — APPROVED** [review]
clgtm, unit test added

**Comment by shwstppr** [comment]
@blueorangutan package

**Review by vishesh92 — APPROVED** [review]
clgtm

**Comment by shwstppr** [comment]
@blueorangutan test

**Review by rohityadavcloud — APPROVED** [review]

**Pull request merged**